### DISCUSSION TITLE
Download problems with release tarballs

### DISCUSSION THREAD

**Discussion by phlax** [community/community#65924] [Repositories]
### Select Topic Area

Bug

### Body

We use bazel which tends to pull source tarchives from Github

We see endemic problems with this - mostly client timeouts

Today this seems to have started failing completely - any CI that tries to pull these tarchives from Github seem to be failing - either timing out, or receiving corrupted files

I have tested this locally with `curl` and im seeing similar things - it seems to have changed now (since earlier "actions" problem was resolved) - essentially:

- correct file (very rare)
- download succeeeds with 200 but tarchive is truncated (ie not valid tar)
- download succeeeds, tarchive appears valid, but contents are corrupt

certain repos appear to fail more than others - im guessing related to file size - but https://github.com/unicode-org/icu/ is definitely the worst

altho the symptoms seem to have changed a little (now seems to timeout after ~30MB, hang and then close without completion), our CI has been broken with this problem all day

**Comment by naldz** [comment]
We are experiencing this as well.

**Comment by hivokas** [comment]
Also having an issue
